### DISCUSSION TITLE
自分の記事を見る時、更新した時、ページ・ビューのカウントアップをさせたくないで、可能ですか？

### DISCUSSION THREAD

**Discussion by oci-tech** [increments/qiita-discussions#256] [要望・アイデア・不具合]
ページ・ビューのカウントは、どのぐらいの読者があるのを把握できるので、重要の機能です。
今のカウント方法は、自分か他人か区別しないようです。
自分の記事を更新した時点もカウントアップされます。これもほしくないです。

多分、区別しなくてよいと思う方もいらっしゃいます。
もし可能であれば、ログイン中の状態で、自分の記事を閲覧した時、更新した時、カウントアップしないという選択肢を増やしてほしいです。

**Comment by tanaka-asami415** [comment]
@liu-wei1 
ご意見ありがとうございます🙏
直近の開発優先度などから考えると、かなり先になってしまうかもしれませんが、検討させていただきます！
